algorithmic_efficiency/workloads/cifar/cifar_pytorch/workload.py

```diff
@@ -103,17 +103,26 @@ def init_model_fn(
     """Dropout is unused."""
     del dropout_rate
     del aux_dropout_rate
+
+    if hasattr(self, '_model'):
+      if isinstance(self._model, DDP) or isinstance(self._model,
```

Comment:
Nit: we can write this more compactly as `isinstance(self._model, (DDP, torch.nn.DataParallel))`

Reply:
Fixed!